cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -124,10 +125,23 @@ public void updateDependencies() {
                 .clearDependencies()
                 .addDependentOn(eqModelId);
 
-        getExportedSVModel()
-                .clearDependencies()
-                .addDependentOn(getExportedTPModel().getId())
-                .addDependentOn(getExportedSSHModel().getId());
+        getExportedSVModel().clearDependencies();
+        List<String> tpIds = legacyIdsForSvDependencies.get(CgmesSubset.TOPOLOGY);
```

Comment:
I understand your solution but I was thinking of something different:
- No properties anymore ;
- We keep information in DependentOn fields ;
- I was thinking about a simple parameter at export that will just say if we export using the extension entirely defined by the user in SV part, meaning that we trust the user. If not, we are allowed to clear the dependencies of SV in extension and to use the other models' dependencies (TP and SSH).

Reply:
That was the second option we discussed, but it will require some changes in the code where the issue was raised. I first wanted to provide a fix that did not required changes in the code already written by the users.

Additionally, building the extension for SV model entirely from scratch requires the user to know some information that now is managed by the CGMES export in a transparent way (mainly, the profile URLs, that are different for each version of CGMES exported).

Reply:
Anyway, I am already preparing a PR that lets the user give the metadata models as inputs, as you suggest.

Reply:
So, with the option relying on a parameter, we must keep the properties?

Reply:
#2993 implements your proposal about respecting the info about dependencies present in the extension. It requires that the external user builds the metadata model in the extension. An example of this, following the same coding scheme seen in the software that raised the issue, can be found in the unit test `LegacyCommonGridModelExportTest::buildSvDependenciesManaginMetadataModels`.

I would keep (for now) the two ways of solving the problem:

- Accept updated dependencies through the legacy mechanism of properties (so the external users do not have to change their code).
- Let the user manage dependencies through metadata model (with an example provided in our code).

Both PRs can be merged  without conflicts. #2993 is based on this one and can be merged after it.

Reply:
Properties management should be removed for next version, here is only to ease transition.